src/Build/ElementLocation/ElementLocation.cs

```diff
@@ -80,7 +80,7 @@ public string LocationString
         /// It is to be used for the project location when the project has not been given a name.
         /// In that case, it exists, but can't have a specific location.
         /// </summary>
-        internal static ElementLocation EmptyLocation
+        public static ElementLocation EmptyLocation
```

Comment:
Why the change from `internal` to `public` here?

Reply:
So that custom analyzers authors would be able to express the unknown location.